src/settings-ui/Settings.UI/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -2,52 +2,55 @@
     x:Class="Microsoft.PowerToys.Settings.UI.Controls.SettingsPageControl"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
+    xmlns:converters="using:CommunityToolkit.WinUI.UI.Converters"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Controls"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     Loaded="UserControl_Loaded"
-    xmlns:converters="using:CommunityToolkit.WinUI.UI.Converters"
-    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
     mc:Ignorable="d">
 
     <UserControl.Resources>
-        <converters:DoubleToVisibilityConverter x:Name="doubleToVisibilityConverter" GreaterThan="0" TrueValue="Visible" FalseValue="Collapsed" />
+        <converters:DoubleToVisibilityConverter
+            x:Name="doubleToVisibilityConverter"
+            FalseValue="Collapsed"
+            GreaterThan="0"
+            TrueValue="Visible" />
     </UserControl.Resources>
 
-    <Grid RowSpacing="24" Padding="20,0,0,0">
+    <Grid Padding="20,0,0,0" RowSpacing="24">
```

Comment:
What guideline (or decision) are these numbers based on? And why not do `Padding="20,0"` ? (to keep layout symmetrical)